ui/app/pages/home/home.component.js

```diff
@@ -280,6 +285,13 @@ export default class Home extends PureComponent {
           {!swapsWelcomeMessageHasBeenShown && swapsEnabled && isMainnet ? (
             <SwapsIntroPopup onClose={setSwapsWelcomeMessageHasBeenShown} />
           ) : null}
+          {(swapsWelcomeMessageHasBeenShown || !swapsEnabled) &&
```

Comment:
The idea is to show the mobile modal *if* the user has already seen the swaps modal -- swaps should be priority.

Reply:
I was thinking about this... I think by the time we ship this to prod (likely after thanksgiving week), we might be able to remove the swap notification. It will have been there for almost 6 weeks, so enough time for most users to see it.

Reply:
With crypto and DeFi still booming, I think we may want to keep the swaps modal our priority.  @jacobcantele should probably make this call.

Reply:
With the way this logic is coded out now, the user gets hit with back-to-back swaps and then mobile popups.  We need a plan for how and when to roll this out.